forum/views.py

```diff
@@ -47,14 +47,14 @@ def get(self, request, sem_name, year):
     def post(self, request, sem_name, year):
         student = Student.objects.get(user=request.user)
         semester = Semester.objects.get(name=sem_name, year=year)
-        transcript = Transcript.objects.get(owner=student, semester=semester)
+        transcript = Transcript.objects.get(owner=request.data['owner'], semester=semester)
```

Comment:
How do you see this working? student is a Student model, but `request.data['owner']` is a String

Reply:
Ah, good catch! I will change it to get a student model given a requested owner_jhed string, and use that student model as the owner object.